app/controllers/reports/parked_clients_controller.rb

```diff
@@ -7,20 +7,43 @@
 module Reports
   class ParkedClientsController < ApplicationController
     before_action :authenticate_user!
+    before_action :set_routes
+    before_action :set_route
 
     def index
-      if can_view_all_clients?
-        clients = parked_scope
-      else
-        clients = parked_scope.non_confidential.full_release
-      end
+      @active_tab = params[:tab] || route_to_html_id(@routes.first)
+      clients = parked_scope
+      clients = clients.non_confidential.full_release unless can_view_all_clients?
+
       @clients = clients.
-        order(prevent_matching_until: :asc).
+        joins(:unavailable_as_candidate_fors).
+        order('unavailable_as_candidate_fors.expires_at asc').
```

Comment:
Add `uacf_t` to `ArelHelper` and replace the string with `uacf_t[:expires_at].asc`?